### PR TITLE
Godot Skeleton v1.1.0

### PR DESCRIPTION
**Pull request by brettchalupa**
I'm working on the next release of Godot Skeleton--v1.1.0. The PR will be in draft mode while I'm working on it, but I figure it might be fun to follow along. Goals are mostly to improve what's already there. No major overhauls. See commits for more details.

### CODE REVIEW

File: sound.gd [hunk 1]
```diff
@@ -0,0 +1,16 @@
+extends Node
```
Comment (jeff-kwak) [inline sound.gd]: just curious; why not use `class_name Sound` (or similar) here.

Comment (brettchalupa) [reply sound.gd]: Good feedback, I'll add that! That'll make it more clear. Godot automatically makes it work when setting up the autoload singleton, but I like the idea of it being named.

Comment (brettchalupa) [reply sound.gd]: I dropped this in, `class_name Sound extends Node` based on your feedback, and it returns this error:

> res://sound.gd:1 - Parse Error: Class "Sound" hides an autoload singleton.

Kind of interesting that it doesn't let ya' do it for autoload singletons. Good learning! :smile:

**Comment by brettchalupa** [comment]
I'm feeling pretty good about these changes. Going to merge this into `main` so its the new default version. Will work on v1.2 after I finish my next game, where I'm sure I'll learn more and merge it back into this.

**Pull request merged**